kolibri/plugins/learn/assets/src/vue/content-page/index.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+
+  <div>
+
+    <breadcrumbs
+      v-if="$route.name === 'explore-content'">
```

Comment:
I hate this. It seems so unVue. Why can't the router pass in parameters into the components as props?!

(Not your fault, just a rant against the design of Vue Router)